src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/PrisonApiClient.kt

```diff
@@ -110,12 +110,12 @@ class PrisonApiClient(
     }
   }
 
-  fun retrieveAgency(agencyId: String): Mono<Agency> {
+  fun retrieveAgency(agencyId: String, activeOnly: Boolean = false): Mono<Agency> {
     val responseType = object : ParameterizedTypeReference<Agency>() {}
```

Comment:
I think the new endpoint returns a `longDescription` instead of a `formatedDescription`, so the type definition will need to change the field name in order to capture the return value - unless I've missed some automagic going on somewhere?